Firestore/Source/Remote/FSTOnlineStateTracker.mm

```diff
@@ -15,27 +15,38 @@
  */
 
 #import "Firestore/Source/Remote/FSTOnlineStateTracker.h"
+
+#include <chrono>  // NOLINT(build/c++11)
+
 #import "Firestore/Source/Remote/FSTRemoteStore.h"
-#import "Firestore/Source/Util/FSTDispatchQueue.h"
 
+#include "Firestore/core/src/firebase/firestore/util/executor.h"
 #include "Firestore/core/src/firebase/firestore/util/hard_assert.h"
 #include "Firestore/core/src/firebase/firestore/util/log.h"
 
+namespace chr = std::chrono;
 using firebase::firestore::model::OnlineState;
+using firebase::firestore::util::AsyncQueue;
+using firebase::firestore::util::DelayedOperation;
+using firebase::firestore::util::TimerId;
 
 NS_ASSUME_NONNULL_BEGIN
 
+namespace {
+
 // To deal with transient failures, we allow multiple stream attempts before giving up and
 // transitioning from OnlineState Unknown to Offline.
 // TODO(mikelehen): This used to be set to 2 as a mitigation for b/66228394. @jdimond thinks that
 // bug is sufficiently fixed so that we can set this back to 1. If that works okay, we could
 // potentially remove this logic entirely.
-static const int kMaxWatchStreamFailures = 1;
+const int kMaxWatchStreamFailures = 1;
 
 // To deal with stream attempts that don't succeed or fail in a timely manner, we have a
 // timeout for OnlineState to reach Online or Offline. If the timeout is reached, we transition
 // to Offline rather than waiting indefinitely.
-static const NSTimeInterval kOnlineStateTimeout = 10;
+const AsyncQueue::Milliseconds kOnlineStateTimeout = chr::seconds(10);
```

Comment:
This is fine, but my initial reaction was to think there was a bug. You're assigning a 'seconds' value to a variable of type 'millis'. It seems like you'd be off by 1000x. In reality, AsyncQueue::Milliseconds is actually a chrono::duration, so this all works fine.

Not relevant to this PR, but what are your thoughts on changing AsyncQueue::Milliseconds to AQ::Duration? That would change this line to `c AQ::Duration timeout = chr::seconds(10);` which reads nicer to me. It would also change `AQ::EnqueueAfterDelay(Milliseconds delay, ...)` to `AQ::EnqueueAfterDelay(Duration delay, ...)`

Reply:
Discussed offline, to briefly recap: my concern is that this would make a constructor like `AQ::Duration foo{100}` non-obvious. `<chrono>` is very reasonable with its conversions: conversions from larger to smaller units are implicit, but the reverse requires `chrono::duration_cast`. It's a compromise, but I'm inclined to leave as is.